apps/aehttp/src/swagger/swagger_internal_handler.erl

```diff
@@ -136,6 +224,24 @@ allowed_methods(Req, State) ->
     }.
 
 
+
```

Comment:
Lots of autogenerated newlines?

Reply:
We could patch the upstream template not to produce them